packages/react-scripts/backpack-addons/cssModules.js

```diff
@@ -9,6 +9,7 @@ const cssModulesEnabled = bpkReactScriptsConfig.cssModules !== false;
 // Backpack / saddlebag node module regexes
 const backpackModulesRegex = /node_modules[\\/]bpk-/;
 const scopedBackpackModulesRegex = /node_modules[\\/]@skyscanner[\\/]bpk-/;
+const backpackSinglePackageModulesRegex = /node_modules[\\/]@skyscanner[\\/]backpack-web/;
```

Comment:
Can you add in the PR description the reason for the changes for backpack single package.

In particular, I'm a little confused how this is currently working as the single package has already been rolled out

Reply:
Hey @runmoore it works currently as per this

https://github.com/Skyscanner/backpack-react-scripts/pull/157#issuecomment-1424027837

Reply:
Specifically when we did the automated rollout we manually added this to everyones BRS config, so this removes the need for people using BRS to do this and anyone trying out spinning up mini POC learning Backpack projects :) 

 ```
  "backpack-react-scripts": {
     "babelIncludePrefixes": [
       "@skyscanner/backpack-web"
     ]
   },
 ```